client/src/features/CampaignDialog.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from 'react';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import { getJWT } from '../utils';
+
+
+export default function CampaignDialog(props) {
+    const { open, onClose} = props;
+    const [name, handleName] = useState("");
+  
+    const handleClose = () => {
+      onClose();
+    }; 
+  
+    const handleCreate = async () => {
+  
+      const data = {
+        'name': name
+      };
+      
+      try {
+        const response = await fetch('http://localhost:5000/campaigns', {
```

Comment:
mentioned this in another PR - don't need the localhost:5000 due to the proxy and we will be using one util to send api requests with authorization